docs/api-guide/serializers.md

```diff
@@ -1181,3 +1181,5 @@ The [drf-writable-nested][drf-writable-nested] package provides writable nested
 [drf-serializer-extensions]: https://github.com/evenicoulddoit/django-rest-framework-serializer-extensions
 [djangorestframework-queryfields]: http://djangorestframework-queryfields.readthedocs.io/
 [drf-writable-nested]: http://github.com/beda-software/drf-writable-nested
+[required-by-default-boolean-field]: http://notes.noumenal.es/2017/12/13/required-by-default.html
```

Comment:
I'd rather see us link to this kind of thing in advanced sections of documentation. Not likely to be the right thing to point users at in this particular case.

Reply:
Here "Customizing field mappings" is already advanced no?

> "Normally if a ModelSerializer does not generate the fields you need by default then you should either add them to the class explicitly, or simply use a regular Serializer class instead. However..." — Just above

It's the sort of thing people struggle to do.